auv_sim/auv_sim_bringup/launch/start.launch

```diff
@@ -1,30 +1,29 @@
 <?xml version="1.0"?>
 <launch>
+  <env name="GAZEBO_MODEL_PATH" value="$(find auv_sim_description)/models:${GAZEBO_MODEL_PATH}"/>
   <arg name="namespace" default="taluy" />
   <arg name="control_rate" default="20.0" />
-  <arg name="use_ned_frame" default="false" />
   <arg name="reference_frame" value="world" />
-
   <arg name="use_gui" default="true" />
   <arg name="debug" default="false" />
-  <arg name="world_name" default="$(find auv_sim_description)/worlds/seabed.world" />
-  <arg name="velocity_control" default="true" />
-  <arg name="joy_id" default="0" />
+  <arg name="world" default="pool" />
+  <arg name="use_ned_frame" default="false" />
+
 
-  <arg name="x" value="9.0" />
-  <arg name="y" value="-3.0" />
-  <arg name="z" value="-87.0" />
+  <!--Initial robot poisition and orientation-->
+  <arg name="x" value="0.0" />
+  <arg name="y" value="0.0" />
+  <arg name="z" value="-1.50" />
+  <arg name="z_sea" value="-87" if="$(eval arg('world') == 'seabed')" />
   <arg name="roll" value="0" />
   <arg name="pitch" value="0" />
   <arg name="yaw" value="-1.8" />
 
-  <include file="$(find auv_sim_bringup)/launch/inc/start_world.launch.xml">
-    <arg name="gui" default="$(arg use_gui)" />
-    <arg name="debug" default="$(arg debug)" />
-    <arg name="world_name" default="$(arg world_name)" />
-    <arg name="namespace" default="$(arg namespace)" />
-    <arg name="velocity_control" default="$(arg velocity_control)" />
-    <arg name="joy_id" default="$(arg joy_id)" />
+
+  <include file="$(find auv_sim_bringup)/launch/inc/start_$(arg world)_world.launch.xml">
```

Comment:
I believe this is an elegant way to select worlds:  
- Naming format for the world launch files: `start_WORLD_launch.xml`  
- Insert the world argument, and done.